helm_deploy/values-dev.yaml

```diff
@@ -14,6 +14,7 @@ generic-service:
     HMPPS_AUTH_URL: https://sign-in-dev.hmpps.service.justice.gov.uk
     PROBATION_INTEGRATION_API_URL: https://external-api-and-delius-dev.hmpps.service.justice.gov.uk
     SPRING_PROFILES_ACTIVE: dev
+    SENTRY_ENVIRONMENT: dev
```

Comment:
Worth adding this for preprod and prod too?

Reply:
was going to confirm all works ok in dev and then update preprod and prod